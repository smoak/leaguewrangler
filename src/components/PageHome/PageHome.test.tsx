import { ShallowWrapper, shallow } from 'enzyme';
import React from 'react';

import PageHomeWithStyles, { PageHome } from './PageHome';

describe('PageHome', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<PageHome classes={{ main: 'main' }} />);
  });

  it('renders', () => {
    expect(component).toMatchSnapshot();
  });
});

describe('PageHomeWithStyles', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<PageHomeWithStyles />);
  });

  it('renders', () => {
    expect(component).toMatchSnapshot();
  });
});
