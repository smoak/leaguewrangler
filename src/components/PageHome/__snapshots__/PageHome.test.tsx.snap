// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHome renders 1`] = `
<div
  className="main"
>
  <Apollo(EventList) />
</div>
`;

exports[`PageHomeWithStyles renders 1`] = `
<PageHome
  classes={
    Object {
      "main": "PageHome-main-1",
    }
  }
/>
`;
